test/puppetlabs/puppetdb/integration/install_gems.clj

```diff
@@ -0,0 +1,30 @@
+(ns puppetlabs.puppetdb.integration.install-gems
+  (:require [clojure.java.io :as io]
+            [puppetlabs.puppetserver.cli.subcommand :as cli]
+            [puppetlabs.services.jruby.jruby-puppet-core :as jruby-puppet-core]
+            [puppetlabs.services.jruby-pool-manager.jruby-core :as jruby-core]
+            [clojure.string :as string]
+            [environ.core :refer [env]]))
```

Comment:
Yep, looks like that should be fine (random aside: had no idea the (exec string) method (as compared to the string[] varieties) behaves the way it does (tokenizing on whitespace)).